diff --git a/lib/index.d.ts b/lib/index.d.ts
index 93101e80bd41308b046cba6fa6d92506cd480a3d..c5221798011fc8efa661fd2f18321108dc7c32d7 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -46,7 +46,7 @@ declare function isLiteral<T extends number>(expected: T): StrictValidator<unkno
 declare function isLiteral<T extends string>(expected: T): StrictValidator<unknown, T>;
 declare function isLiteral<T>(expected: T): StrictValidator<unknown, T>;
 declare const isString: () => StrictValidator<unknown, string>;
-declare function isEnum<T extends boolean | string | number | null>(values: T[]): StrictValidator<unknown, T>;
+declare function isEnum<T extends boolean | string | number | null>(values: ReadonlyArray<T>): StrictValidator<unknown, T>;
 declare function isEnum<T>(enumSpec: Record<string, T>): StrictValidator<unknown, T>;
 declare const isBoolean: () => StrictValidator<unknown, boolean>;
 declare const isNumber: () => StrictValidator<unknown, number>;
